core/drivers/imx/mu/imx_mu_8ulp.c

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022 NXP
+ */
+#include <drivers/imx_mu.h>
+#include <initcall.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <mm/core_memprot.h>
+
+#include "imx_mu_common.h"
+
+#define MU_TCR		  0x120
+#define MU_TSR		  0x124
+#define MU_RCR		  0x128
+#define MU_RSR		  0x12C
+#define MU_TRn(n)	  (0x200 + 0x4 * (n))
+#define MU_RRn(n)	  (0x280 + 0x4 * (n))
+#define MU_TSR_TEn(n)	  SHIFT_U32(0x1, (n))
+#define MU_RSR_RFn(n)	  SHIFT_U32(0x1, (n))
```

Comment:
`BIT(n)`?

By the way, really prefer full uppercase names for macros. Could these be:
```c
#define MU_TSR_TE(n)	  BIT32(n)
#define MU_RSR_RF(n)	  BIT32(n)
```
?

Reply:
That makes sense